### PR TITLE
analog: add optional data parameter to GetParameter command

### PR DESCRIPTION
**Pull request by DeadByElpy**

### CODE REVIEW

File: src/analog/CommandBinaryBuffer.ts [hunk 1]
```diff
@@ -413,6 +426,10 @@ type TParameterData =
     IParameterPulseChannelsScanConfig |
     IParameterPulseChannelsSetConfig;
 
+/* sorted by parameter id */
+type TRequestParameterData =
+    IRequestParameterAbsoluteDataMC;
```
Comment (DarkPark) [inline src/analog/CommandBinaryBuffer.ts:431]: do we need this alias?

maybe in one line?

File: src/analog/commands/downlink/GetParameter.ts [hunk 2]
```diff
@@ -1,33 +1,35 @@
...
 const COMMAND_ID = 0x04;
-const COMMAND_BODY_SIZE = 1;
 
 const examples: TCommandExampleList = [
     {
         name: 'request absolute data (not multichannel device)',
-        parameters: {id: deviceParameters.ABSOLUTE_DATA},
+        parameters: {
+            id: deviceParameters.ABSOLUTE_DATA,
+            data: undefined
```
Comment (DarkPark) [inline src/analog/commands/downlink/GetParameter.ts:14]:
```suggestion
            data: null
```

?

this field may be not visible after serialization in jooby-gui

File: src/analog/commands/downlink/GetParameter.ts [hunk 3]
```diff
@@ -68,17 +70,18 @@ class GetParameter extends Command {
...
-        return new GetParameter({id: buffer.getUint8()});
+        return new GetParameter(buffer.getRequestParameter());
     }
 
     // returns full message - header with body
-    // eslint-disable-next-line class-methods-use-this
     toBytes (): Uint8Array {
-        const buffer = new CommandBinaryBuffer(COMMAND_BODY_SIZE);
+        const size = CommandBinaryBuffer.getRequestParameterSize(this.parameters);
+        console.log({size});
```
Comment (DarkPark) [inline src/analog/commands/downlink/GetParameter.ts:79]:
```suggestion
```

**Review by DarkPark — CHANGES_REQUESTED** [review]

File: src/analog/commands/downlink/GetParameter.ts (hunk 3 above)
Comment (DeadByElpy) [reply src/analog/commands/downlink/GetParameter.ts:79]: fixed

File: src/analog/commands/downlink/GetParameter.ts (hunk 2 above)
Comment (DeadByElpy) [reply src/analog/commands/downlink/GetParameter.ts]: null added as possible value and by default

File: src/analog/CommandBinaryBuffer.ts (hunk 1 above)
Comment (DeadByElpy) [reply src/analog/CommandBinaryBuffer.ts]: TRequestParameterData removed

**Review by DarkPark — APPROVED** [review]

**Pull request merged**